### ISSUE TITLE
where is the attention mechanism in your code?

### ISSUE THREAD

**Issue by StackChan** [zccqq/stMDA#2]
"Then, stMDA uses attention mechanism to obtain the joint representation" in your paper
from https://academic.oup.com/view-large/figure/462533777/bbae257f1.tif.

But where is the attention mechanism in your code?

**Comment by StackChan** [comment]
Also,
`        contrast_loss = torch.norm(inference_outputs['x1'] - inference_outputsG['x1'])
        contrast_loss += torch.norm(inference_outputs['qz'].loc - inference_outputsG['qz'].loc)
        contrast_loss += torch.norm(inference_outputs['x1'] - inference_outputsI['x1'])
        contrast_loss += torch.norm(inference_outputs['qz'].loc - inference_outputsI['qz'].loc)
        loss = reconst_loss + kl_loss / 3 + contrast_loss / 4
`
It's different from  your saying "(D) The spatial distribution alignment improves the consistency of the spatially local and global distribution of each spot in the modality-specific representations using MMD. " ( your contrast method is different from MMD)
